package.json

```diff
@@ -55,7 +56,8 @@
     "typescript": "^2.9.2"
   },
   "peerDependencies": {
-    "@angular-devkit/architect": ">=0.7.0-rc.0",
-    "@angular-devkit/build-angular": ">=0.7.0-rc.0"
+    "@angular-devkit/architect": ">=0.7.0-rc.3",
```

Comment:
It's preferable to have the least possible version as a dependency. Are you confident `rc.3` is the minimal required version?

Reply:
Yes. It needs this commit, which was introduced with rc.3:

https://github.com/angular/angular-cli/commit/cb588b68b34641cbfc31b563f741f94f48211b2d#diff-6c0b312b8bd33db747e50549a66f7ab1